README.md

```diff
@@ -142,6 +142,15 @@ MSG_FLAGS = {
 
 # use this app to open url when there are multiple
 URL_VIEW = 'urlview'
+
+# Specifies range of colors to use for drawing different users
+# - 1, 8, 16, 256: range of colors to use
+# 233 is recommended and default, because it excludes gray
+USERS_COLORS = tuple(range(233))
```

Comment:
let's use 16 colors by default. because it looks good both for light & dark terminal. 233 looks bad on light theme